lib/routes/steamcommunity/namespace.ts

```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```

Comment:
Please use `steampowered` instead.

Reply:
I checked the secondary domain was `steamcommunity`. Should I move it to the route folder `steam` or change the namespace?

Reply:
Move to `steam`.